architecture/core_concepts/pods_and_services.adoc

```diff
@@ -38,6 +38,11 @@ maintain state when recreated. Therefore pods should usually be managed by
 higher-level link:deployments.html#replication-controllers[controllers],
 rather than directly by users.
 
+[IMPORTANT]
+====
+The maximum supported number of pods within an {product-title} cluster is 110.
```

Comment:
@timothysc The max-pods setting is talked about here: https://docs.openshift.com/enterprise/3.1/admin_guide/manage_nodes.html#configuring-node-resources

I think adding a section about over-riding this might be out of scope for this BZ, but we can make another and work on it. 

Also, I considered changing the language to say "recommended minimums" instead of "supported minimums" but that might be against the idea of this BZ. I don't think we can have both... @timothysc @jeremyeder Do you have thoughts on this? Which to prefer?